packages/functional-tests/tests/react-conversion/signup.spec.ts

```diff
@@ -64,13 +64,13 @@ test.describe('severity-1 #smoke', () => {
       await settings.signOut();
     });
 
-    // TODO this test is skipped until we sort out
-    // how to record an email bounce for testing
-    test.skip('signup, bounce email', async ({
+    test('signup, bounce email', async ({
       page,
       target,
       pages: { login, signupReact },
     }) => {
+      test.skip(true, 'this test is skipped until we sort out how to record an email bounce for testing');
```

Comment:
🤞 Hopefully this works. Changed from `test.skip()` above on L67 to an inline `test.skip()` here with the TODO comment.